cfme/configure/access_control.py

```diff
@@ -232,17 +257,65 @@ def copy(self):
         assert view.is_displayed
         return new_user
 
-    def delete(self, cancel=True):
-        """ Delete existing user
-            Args:
-                cancel: Default value 'True', user will be deleted
-                        'False' - deletion of user will be canceled
+    def _delete_using_all_selection(self):
         """
-        view = navigate_to(self, 'Details')
-        view.configuration.item_select('Delete this User', handle_alert=cancel)
+        Delete existing user by selecting it from the access control group list
+
+        Throws:
+            RBACOperationBlocked: If operation is blocked due to current user
+                not having appropriate permissions OR delete is not allowed
+                for currently selected user
+        """
+        name_column = "Full Name"
+        find_row_kwargs = {name_column: self.name}
+
+        view = navigate_to(User, 'All')
+        try:
+            user_row = view.paginator.find_row_on_pages(view.table, **find_row_kwargs)
+            # Zero index is the unnamed column with the checkbox
+            user_row[0].check()
+        except NoSuchElementException:
+            raise Exception("Failed to find user row with name {}".format(self.name))
+
+    def delete(self, cancel=True, all_selection=False):
+        """
+        Delete existing user
+        Args:
+            cancel: Default value 'True', user will be deleted
+                    'False' - deletion of user will be canceled
+        Throws:
+            RBACOperationBlocked: If operation is blocked due to current user
```

Comment:
**QUESTION**: Given the above statement, it **could** just raise a normal Exception instead.